src/ips/messages.ts

```diff
@@ -17,18 +17,18 @@ export class Messages {
 
   private readonly get = (action: AllActions, ipsClass: Classes, messageId: string): string => {
     const header: string = this.wsmanMessageCreator.createHeader(action, `${this.resourceUriBase}${ipsClass}`, messageId)
-    const body: string = this.wsmanMessageCreator.createBody(Methods.GET)
+    const body: string = this.wsmanMessageCreator.createCommonBody(Methods.GET)
     return this.wsmanMessageCreator.createXml(header, body)
   }
 
   private readonly put = (action: AllActions, ipsClass: Classes, messageId: string, data: OptInServiceResponse): string => {
     const header: string = this.wsmanMessageCreator.createHeader(action, `${this.resourceUriBase}${ipsClass}`, messageId)
-    let body: string = this.wsmanMessageCreator.createPutBody(data)
-    body = body.replace(`<r:${Classes.IPS_OPT_IN_SERVICE}>`, `<r:IPS_OptInService xmlns:r="${this.resourceUriBase}${Classes.IPS_OPT_IN_SERVICE}">`)
+    const key = Object.keys(data)[0]
```

Comment:
is this guaranteed to always be the zero index? what if there are no keys? or multiple keys?